### DISCUSSION TITLE
Organized by events ID module

### DISCUSSION THREAD

**Discussion by admarnelson** [Security-Onion-Solutions/securityonion#7191] [Ideas] [answered]
I was thinking. it was very good if has EVENT ID Module (all events from hosts and network in one modules). I mean:
when we are hunting for threat analysts looking for network and hosts events ID, hash value to get better understand what's up and getting a module related events ID will help quick hunting and correlate with diferents modules or value. 

Best regards team

AdmarNelson 
CyberAnalyst SOC

**Comment by dougburks** [comment, marked as answer]
I'm not sure I understand what you're asking. Please provide more detail and an example or screenshots if possible.

**Reply by admarnelson** [reply]
Windows Event ID Module, if agents are installed. it helps us hunting. sorry my bad English

![image](https://user-images.githubusercontent.com/93060289/154009192-475eaa00-b8f9-40fa-99f0-bfba5418a3d0.png)

**Reply by dougburks** [reply]
If you are collecting Windows event logs, then the Windows Event ID should be available in the `event.code` field:
https://www.elastic.co/guide/en/ecs/current/ecs-event.html#field-event-code

Here's an example from our Hunt interface:
![image](https://user-images.githubusercontent.com/1659467/154053731-18f3c549-52a2-4cb7-bf00-a85204bc4213.png)
